src/main/java/com/gluonhq/strange/demo/Demo.java

```diff
@@ -46,11 +46,7 @@
     public static void main(String[] args) throws ExecutionException, InterruptedException {
         System.out.println("Hello, demo");
         Program p = new Program(4);
-        Step s = new Step();
```

Comment:
For a demo, it's easier to explain this step by step

Reply:
Done